packages/example/lib/api.ts

```
@@ -0,0 +1,23 @@
+const DEFAULT_API = 'https://api.onekeytest.com'
+
+function getApiKey(apiKey:string, fromBase64:boolean = true) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)